src/renderer/components/siteinfotools/SnapshotsTableList.tsx

```diff
@@ -171,6 +181,7 @@ const renderCellMoreMenu = (snapshot: BackupSnapshot, site: Site, provider: HubP
 		<FlyDropdown
 			caret={false}
 			className={styles.SnapshotsTableList_MoreDropdown}
+			classNameListItem={offline ? styles.DropdownItemOffline : ''}
```

Comment:
Similar to above: 

```suggestion
			classNameListItem={classnames([styles.DropdownItemOffline]: offline )}
```

Reply:
Ooo that's fancy, thanks for catching that!